### PR TITLE
Mm/init no docs

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
Allow the user to skip sending the docs from a starter by passing a `--no-docs` flag
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/init/src/index.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ async function init() {
   const isTrackingAvailable =
     findArgument(process.argv, "tracking") !== "false";
   const maybeRepositorySubdomain = findArgument(process.argv, "repository");
+  const sendDocs = findArgument(process.argv, "no-docs") ? true : false;
```
Comment (Heyrows) [inline packages/init/src/index.ts]: I don't think that would work, did you tested it ?
If I look at the findArgument function it expects a value after that.

Maybe we could make another helper like "findFlag"

**Review by chengoconnell — APPROVED** [review]

File: packages/init/src/steps/sendStarterData.ts [hunk 2]
```diff
@@ -31,5 +34,10 @@ export async function sendStarterData(
 
   await sendCustomTypesFromStarter(repository, authTokenFromCookie, base, cwd);
 
+  if (sendDocs === false) {
```
Comment (Heyrows) [inline packages/init/src/steps/sendStarterData.ts:37]: If the starter has documents but we don't want to send them, i'm guessing they have another use, do we want to delete them ?

From a user point of view if he/she puts the flag, I would expect my folder to be untouched i think.

Comment (MarcMcIntosh) [reply packages/init/src/steps/sendStarterData.ts:37]: I asked come about this yesterday and we want to delete them

File: packages/init/src/index.ts (hunk 1 above)
Comment (MarcMcIntosh) [reply packages/init/src/index.ts]: I'll take a look, and nope I've not had the chance to run because of all the setup required. good catch though :)

Comment (MarcMcIntosh) [reply packages/init/src/index.ts]: I've run it now and it deletes the documents, just fine :)

File: packages/init/src/steps/sendStarterData.ts (hunk 2 above)
Comment (Heyrows) [reply packages/init/src/steps/sendStarterData.ts:37]: Ok :)

**Review by Heyrows — APPROVED** [review]

**Pull request merged**